ci/build_website.sh

```diff
@@ -1,20 +1,20 @@
-
+#!/usr/bin/env bash
 echo "Starting build process..."
 
+# Get parameters with defaults 
+VERSION=${VERSION:-${1:-"dev"}}
+LANGUAGE=${LANGUAGE:-${2:-"en"}}
+
+echo "Building version: $VERSION, language: $LANGUAGE"
+
 # Install dependencies
 pip install -r requirements.txt
 
 # Clean tags directory
 rm -rf DISCOVER/_tags/*
 
-echo "Building English version..."
-sphinx-build -b html DISCOVER/ DISCOVER/_build/html
-
-
-# Copy root level files if they exist
-if [ -f "DISCOVER/_static/404.html" ]; then
-  cp DISCOVER/_static/404.html DISCOVER/_build/html/
-fi
 
+echo "Building $LANGUAGE version..."
+WEBSITE_VERSION="$VERSION" WEBSITE_LANGUAGE="$LANGUAGE" sphinx-build -b html DISCOVER/ DISCOVER/_build/html
 
-echo "Build completed successfully"
\ No newline at end of file
+echo "Build completed successfully: $VERSION/$LANGUAGE"
```

Comment:
sphinx finishes its logs with a sentence about the success or failure of the build process. I have just realized this can be confusing for debugging. The build can fail if something gets broken which will mean the logs will end up with `sphinx: build failed due to xyz` and in the following line `Build completed successfully`